samples/bluetooth/fast_pair/input_device/README.rst

```diff
@@ -78,6 +78,9 @@ See :ref:`ug_bt_fast_pair_provisioning` in the Fast Pair user guide for details.
    :start-after: nrf_desktop_fastpair_important_start
    :end-before: nrf_desktop_fastpair_important_end
 
+.. important::
+   To support the input device use case, you must select the ``input device`` option in the **Device Type** field during device model registration.
```

Comment:
Is "input device" a clickable element? If so, it should be represented as ":guilabel:`input device`"

Reply:
Yes, accessed via dropdown list:

![Screenshot from 2024-05-10 11-43-07](https://github.com/nrfconnect/sdk-nrf/assets/17042049/c03b8d7c-1291-44d3-9d74-d033cc41fa67)

Reply:
discussed offline: 

```
you must select :guilabel:`Input Device` option in in the **Device Type** list .....
```